tests/test_text/test_dispersion.py

```diff
@@ -0,0 +1,53 @@
+# tests.test_text.test_freqdist
+# Tests for the frequency distribution visualization
+#
+# Author:   Rebecca Bilbro
+# Github:   @rebeccabilbro
+# Created:  2017-03-22 15:27
+#
+# Copyright (C) 2018
+# For license information, see LICENSE.txt
+#
+# ID: test_freqdist.py [bd9cbb9] rbilbro@districtdatalabs.com $
+
+"""
+Tests for the frequency distribution text visualization
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import sys
+import pytest
+
+from yellowbrick.text.dispersion import *
+from tests.dataset import DatasetMixin
+from tests.base import VisualTestCase
+
+
+##########################################################################
+## DispersionPlot Tests
+##########################################################################
+
+class FreqDistTests(VisualTestCase, DatasetMixin):
+
+    @pytest.mark.xfail(
+        sys.platform == 'win32', reason="images not close on windows"
+    )
```

Comment:
I'd advise leaving this `xfail` off until we know for certain that it won't work on windows. Note if you comment or remove these lines, you'll also need to remove `import sys` to avoid a linter error.